socorro/unittest/external/postgresql/unittestbase.py

```diff
@@ -19,11 +21,15 @@ class PostgreSQLTestCase(TestCase):
     required_config = Namespace()
     required_config.namespace('database')
     required_config.add_option(
+        name='database_url',
+        default='postgres://test:aPassword@localhost:5432/socorro_integration_test',
+        doc='Name of database to manage',
+    )
+    required_config.add_option(
```

Comment:
Using a URL isn't Heroku-specific. I use it whenever I can when I do non-django. The reason I like it is because it files into one environment variable instead of 5. 
Also, wherever you type in `database_password` you'd also type in `database_url`. 

Either way, I'm not convinced. But I am convinced that you know better. :)
My comment was just a question after all :)